cv32/sim/uvmt_cv32/vsim.mk

```diff
@@ -21,64 +21,60 @@
 #
 ###############################################################################
 
-USES_DPI=1
-
-ifeq ($(USES_DPI),1)
-  DPILIB_VLOG_OPT = 
-  DPILIB_VSIM_OPT = -sv_lib $(UVM_HOME)/../../uvm-1.2/linux_x86_64/uvm_dpi
-  DPILIB_TARGET = dpi_lib$(BITS)
-else
-  DPILIB_VLOG_OPT = +define+UVM_NO_DPI 
-  DPILIB_VSIM_OPT = 
-  DPILIB_TARGET =
-endif
-
-LIBDIR  = $(UVM_HOME)/lib
-LIBNAME = uvm_dpi
+GUI         ?= 0
 
+# Questasim commands
 VLIB      = vlib
-VWORK     = work
+VLOG      = vlog
+VOPT      = vopt
+VSIM      = vsim
 
-VLOG          = vlog
-VLOG_FLAGS    = -pedanticerrors -suppress 2577 -suppress 2583 \
-        $(DPILIB_VLOG_OPT) \
-	-timescale "1ns/1ps" \
-        -mfcu \
-        -suppress 2181 \
-	-suppress 13262 \
-        +acc=rb \
-        -writetoplevels  uvmt_cv32_tb \
-        +incdir+$(UVM_HOME)/src \
-        $(UVM_HOME)/src/uvm.sv
+# Work library
+VWORK     = work
 
+# Build parameters
+VLOG_FLAGS    = -pedanticerrors -timescale "1ns/1ps" -mfcu +acc=rb \
```

Comment:
UVM library be added in this command line where QUESTA_HOME is the path to QuestaSim installation.

VLOG_FLAGS    =-L $(QUESTA_HOME)/uvm-1.2 -pedanticerrors -timescale "1ns/1ps" -mfcu +acc=rb \